### ISSUE TITLE
The system encountered an error while loading m3u playlists

### ISSUE THREAD

**Issue by MrksHfmn** [gerbera/gerbera#2487]
I have mounted my complete music directory. among them are also many folders with playlists (.m3u). As soon as Gerbera encounters such a playlist and wants to index it, the software crashes.
```

022-03-14 09:19:49   info: Gerbera UPnP Server 1.10.0 - http://gerbera.io/
2022-03-14 09:19:49   info: Copyright 2016-2021 Gerbera Contributors.
2022-03-14 09:19:49   info: Gerbera is free software, covered by the GNU General Public License version 2
2022-03-14 09:19:49   info: Loading configuration from: /var/run/gerbera/config.xml
2022-03-14 09:19:49   info: Checking configuration...
2022-03-14 09:19:49   info: Configuration check succeeded.
2022-03-14 09:19:49   info: Loading 0 configuration items from database
2022-03-14 09:19:49   info: Loading autoscan location: /content; recursive: true, last_modified: unset
2022-03-14 09:19:49   info: Initialising UPnP with interface: <unset>, port: 49494
2022-03-14 09:19:49   info: IPv4: Server bound to: 192.168.1.100:49494
2022-03-14 09:19:49   info: Will send UPnP Alive advertisements every 60 seconds
2022-03-14 09:19:49   info: The Web UI can be reached by following this link: http://192.168.1.100:49494
2022-03-14 09:21:07   info: Processing playlist: /content/musik/flac/Playlists/Classic Rock/Classic Rock.m3u
```

**Comment by whyman** [comment]
Hey @MrksHfmn!

Can provide a debug for the same? Enable it by passing `--debug` to Gerbera.

**Comment by KarlStraussberger** [comment]
Thank you for reporting.

This issue is already fixed in #2464 and I hope to make the new release by the end of the week.
